src/index.ts

```diff
@@ -104,21 +111,50 @@ const showPrivacyManager = () => {
 	void initialised.then(frameworkCMP?.showPrivacyManager);
 };
 
-export const cmp: CMP = (window.guCmpHotFix.cmp ||= {
-	init,
-	willShowPrivacyMessage,
-	willShowPrivacyMessageSync,
-	hasInitialised,
-	showPrivacyManager,
-	version: __PACKAGE_VERSION__,
-
-	// special helper methods for disabling CMP
-	__isDisabled: isDisabled,
-	__enable: enable,
-	__disable: disable,
-});
+export const cmp: CMP = (() => {
+	if (isServerSide) {
+		return {
+			init: (_: { pubData?: PubData; country?: Country }) => void 0,
+			showPrivacyManager: () => {
+				void 0;
+			},
```

Comment:
just for consistency
```suggestion
			showPrivacyManager: () => void 0,
```